README-ar.md

```diff
@@ -0,0 +1,198 @@
+
+Usage: bfy [OPTIONS] [SOURCE]
+
+Arguments:
+  [SOURCE]
+          The brainfuck source code file to run (if not will be entered in REPL mode)
+
+Options:
+  -f, --features <FEATURES>
+          Possible values:
+          - no-reverse-value:
+            If the value is you want decrement the value and the value is 0, don't set the value to 255, otherwise decrement the value. If the
+            value is you want increment the value and the value is 255, don't set the value to 0, otherwise increment the value. The alias are:
+            `nrv`
+          - reverse-pointer:
+            If the pointer at the end of the array, set the pointer to 0, otherwise increment the pointer. If the pointer at the beginning of the
+            array, set the pointer to the end of the array, otherwise decrement the pointer. The alias are: `rp`
+          - allow-utf8:
+            Allow the use of utf8 characters (32 bit), otherwise only 8 bit characters are allowed. Use this feature with caution because it
+            increases the cell size from 8 bits to 32 bits. It also allow you to use the emoji in your brainfuck code :D, This is if you can
+            preserve your mind so that you can access their digital value :). The `u32` in rust can only store values from 0 to 4294967295, but we
+            can only use 0 to 1114111 (0x10FFFF) for now. The alias are: `utf8`
+
+  -a, --array-size <ARRAY_SIZE>
+          The brainfuck array size
+          
+          [default: 30000]
+
+  -w, --without-tiles
+          Dont print the tiles (e.g. exit code, file name, etc)
+
+  -h, --help
+          Print help information (use `-h` for a summary)
+
+  -V, --version
+          Print version information
+```
+
+</details>
+
+### أمثلة
+
+```bash
+bfy test_code/hello_world.bf
+```
+```text
+Hello world!
+Successfully ran brainfuck source code from file: test_code/hello_world.bf
+Exiting with code: 0
+```
+
+```bash
+bfy -w test_code/hello_world.bf
+```
+```text
+Hello world!
+```
+
+```bash
+bfy test_code/print_hi_yooo.bf
+```
+```text
+Hi yoooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo!Successfully ran brainfuck source code from file: test_code/print_hi_yooo.bf
+Exiting with code: 0
+```
+
+```bash
+bfy -w test_code/print_hi_yooo.bf
+```
+```text
+Hi yoooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo!
+```
+
+```bash
+bfy test_code/like_cat.bf
+```
+![output](./screenshots/like_cat_output.png)
+> ملاحظة: الإخراج ليس هو نفسه الإصدار الحالي للمترجم الفوري ، ولكنه مماثل لإخراج المترجم الفوري عندما أكتب الكود.
+
+## REPL
+```bash
+bfy # REPL mode
+```
+![print @ and A in the repl](./screenshots/repl_print_at_and_A_0.1.0.png)
+
+## لكى يفعل
+- [ ] أضف المزيد من الاختبارات
+- [ ] أضف المزيد من الأمثلة
+- [ ] قم بإنشاء منسق brainfu * k
+- [ ] أضف تمييز بناء الجملة في REPL
+- [ ] أضف الإكمال التلقائي في REPL
+- [ ] دعم ميزة حجم الصفيف الديناميكي
+
+## مصادر
+- [مخنث فى ويكيبيديا](https://en.wikipedia.org/wiki/Brainfuck)
```

Comment:
Please don't translate the language name

Reply:
Do you mean the link title?

Reply:
yep

Reply:
Okay will do!